examples/OpticalTracker/scripts/view.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export DD4hepExamplesINSTALL=`pwd`
+source install/bin/thisOpticalTracker.sh
+prefix=install/examples/OpticalTracker
+
+outputFile=geo.root
+compactFile=$prefix/compact/pfrich.xml
+dd_web_display --export -o $outputFile $compactFile
```

Comment:
Despite its name, `dd_web_display` is not actually a dd4hep tool :-) But it might make sense to upstream it.

Reply:
Ah, I never knew ;)
Well.. is there a more `DD4hep` way to visualize? We could remove this `view.sh` script, but it's nice to have for certain people (me). I'm also in favor of upstreaming `dd_web_display`, given how useful it is for all.

Reply:
The dd4hep way is to use geoDisplay or ddeve. even the dd_web_display we should probably retire and use geoWebDisplay. e.g. see here for an example in tests: https://github.com/AIDASoft/DD4hep/blob/master/examples/ClientTests/CMakeLists.txt#L226